src/ui/Windows/res/PasswordSafe3.rc2

```diff
@@ -1052,5 +1062,9 @@ STRINGTABLE
 BEGIN
 IDS_SCRCAP_TT_DISALLOWED_DEFAULT	"Screen capture protection is on."
 IDS_SCRCAP_TT_ALLOWED_DBPREF		"Screen capture protection is off."
+IDS_SCRCAP_TT_OVERRIDE_INVALID      "Screen capture protection is invalid (failure)."
+IDS_SCRCAP_TT_OVERRIDE_NONE         "Screen capture protection is on."
 IDS_SCRCAP_TT_OVERRIDE_CMDLINE		"Screen capture protection has been overridden by command line."
+IDS_SCRCAP_TT_OVERRIDE_ICA          "Screen capture protection has been overridden by detection of the ICA protocol (i.e., Citrix)."
```

Comment:
Suggest unifying with _RDP into something like "Running under remote desktop, screen capture protection unavailable."

Reply:
How firm are you on the suggestion?

It might be good to express the reason for reducing app security in a user-friendly manner. In this case, if a user seeks tech support, they can convey the message which indicates cause more specifically... but in a user-friendly manner. And the detail might help the user... if a user does not expect Citrix to be running, or RDP to be running, the awareness might be good for security.

Reply:
Discussed above. TL;DR - pretty firm :-)